### PR TITLE
Fix php version 7 in console

### PR DESCRIPTION
**Pull request by mehdi-fathi**
fix issue #10870 for php version 7.

### CODE REVIEW

**Comment by markstory** [comment]
Based on the discussion wouldn't it be better to remove the welcome banner entirely?

File: src/Console/Shell.php [hunk 1]
```diff
@@ -84,6 +84,13 @@ class Shell
     public $OptionParser;
 
     /**
+     * A var for maintain phpversion()
+     *
+     * @var show php version
+     */
+    public $phpVersion;
```
Comment (markstory) [inline src/Console/Shell.php]: We are trying to not add any new public properties. Can this be protected.

Comment (dereuromark) [inline src/Console/Shell.php]: Why is this needed in the first place if the scope is only this welcome method?

File: src/Console/Shell.php [hunk 2]
```diff
@@ -242,12 +249,14 @@ public function startup()
      */
     protected function _welcome()
     {
+        $phpInfo = explode('~', phpversion());
+        $this->phpVersion = (isset($phpInfo[0]) ? $phpInfo[0] : phpversion());
```
Comment (dereuromark) [inline src/Console/Shell.php]: The isset is not necessary as it will always have a 0 element after an explode() of any content input.

File: src/Console/Shell.php (hunk 1 above)
Comment (mehdi-fathi) [inline src/Console/Shell.php]: @dereuromark Maybe this uses in another method later.

File: src/Console/Shell.php (hunk 2 above)
Comment (inoas) [inline src/Console/Shell.php]: I would return the full php version as `phpversion()` returns it anyway. And I do not see the reason to use a variable then.

File: src/Console/Shell.php (hunk 1 above)
Comment (dereuromark) [inline src/Console/Shell.php]: We dont code for maybes usually :)

File: src/Console/Shell.php (hunk 2 above)
Comment (mehdi-fathi) [inline src/Console/Shell.php]: @inoas you can see bin/cake in php 7.x.You should reed #10870

Comment (dereuromark) [inline src/Console/Shell.php]: The consensus was to fix it the proper way by only displaying it alltogether for the mentioned main page and --version --help things. 
This here only fixes a small symptom, so let's please focus on the actual way to solve it right.

**Comment by robertpustulka** [comment]
I don't see how this PR fixes #10870.

File: src/Console/Shell.php (hunk 1 above)
Comment (markstory) [inline src/Console/Shell.php]: I agree, given the usage of this variable it does not require a property at all.

**Comment by dereuromark** [comment]
Resolved with https://github.com/cakephp/cakephp/pull/10879
